vuu-ui/packages/vuu-filters/src/filter-utils.ts

```diff
@@ -1,494 +1,360 @@
-      filter.filters.forEach((f) => clauses.push(...filterClauses(f)));
+      filter.filters.forEach((f) => clauses.push(...filterClauses(f)))
     } else {
-      // TODO why did we originally stringify 'values' ?
-      // clauses.push({ column, op, value: value ?? values?.join(',') });
-      clauses.push(filter);
+      clauses.push(filter)
     }
   }
-  return clauses;
-};
+  return clauses
+}
 
 type AddFilterOptions = {
-  combineWith: FilterCombinatorOp;
-};
+  combineWith: FilterCombinatorOp
+}
 
 const DEFAULT_ADD_FILTER_OPTS: AddFilterOptions = {
-  combineWith: "and",
-};
+  combineWith: 'and',
+}
 
-export function addFilter(
+export const addFilter = (
   existingFilter: Filter | null,
   filter: Filter,
   { combineWith = AND }: AddFilterOptions = DEFAULT_ADD_FILTER_OPTS
-): Filter | null {
+): Filter | null => {
   if (includesNoValues(filter)) {
     if (isMultiClauseFilter(filter)) {
       // TODO identify the column that is contributing the no-values filter
     } else {
       existingFilter = removeFilterForColumn(existingFilter, {
         name: filter.column,
-      });
+      })
     }
   } else if (includesAllValues(filter)) {
-    // A filter that returns all values is a way to remove filtering for this column
     if (isMultiClauseFilter(filter)) {
       // TODO identify the column that is contributing the all-values filter
-    } else {
-      return removeFilterForColumn(existingFilter, { name: filter.column });
     }
+    return removeFilterForColumn(existingFilter, { name: filter.column ?? '' })
   }
 
   if (!existingFilter) {
-    return filter;
-  } else if (!filter) {
-    return existingFilter;
+    return filter
+  }
+  if (!filter) {
+    return existingFilter
   }
-
   if (existingFilter.op === AND && filter.op === AND) {
     return {
       op: AND,
       filters: combine(existingFilter.filters, filter.filters),
-    };
-  } else if (existingFilter.op === AND) {
-    const filters = replaceOrInsert(existingFilter.filters, filter);
-    return filters.length > 1 ? { op: AND, filters } : filters[0];
-  } else if (filter.op === AND) {
-    return { op: AND, filters: filter.filters.concat(existingFilter) };
-  } else if (filterEquals(existingFilter, filter, true)) {
-    return filter;
-  } else if (sameColumn(existingFilter, filter)) {
-    return merge(existingFilter, filter);
-  } else {
-    return { op: combineWith, filters: [existingFilter, filter] };
+    }
+  }
+  if (existingFilter.op === AND) {
+    const filters = replaceOrInsert(existingFilter.filters, filter)
+    return filters.length > 1 ? { op: AND, filters } : filters[0]
+  }
+  if (filter.op === AND) {
+    return { op: AND, filters: filter.filters.concat(existingFilter) }
+  }
+  if (filterEquals(existingFilter, filter, true)) {
+    return filter
   }
+  if (sameColumn(existingFilter, filter)) {
+    return merge(existingFilter, filter)
+  }
+  return { op: combineWith, filters: [existingFilter, filter] }
 }
```

Comment:
Can we get a bit more whitespace? 

```suggestion
  if (filter.op === AND) {
    return { op: AND, filters: filter.filters.concat(existingFilter) }
  }
  
  if (filterEquals(existingFilter, filter, true)) {
    return filter
  }
  
  if (sameColumn(existingFilter, filter)) {
    return merge(existingFilter, filter)
  }
  
  return { op: combineWith, filters: [existingFilter, filter] }
}
```